tests/integration/Core/Framework/Seo/SeoUrlPersisterTest.php

```diff
@@ -480,6 +487,76 @@ public function testUpdateSeoUrlForDifferentSalesChannelsWithSameSeoPathInfo():
         }
     }
 
+    // probably we should add asserts etc to this test case,
+    // right now its just checking that its not crashing in the multilingual case.
+    public function testMultilingualIsolationCase(): void
+    {
+        /** @var EntityRepository<ProductCollection> $productRepository */
+        $productRepository = static::getContainer()->get('product.repository');
+        /** @var EntityRepository<LanguageCollection> $languageRepository */
+        $languageRepository = static::getContainer()->get('language.repository');
+
+        $languageCriteria = (new Criteria())->setLimit(2);
+        $languageIds = $languageRepository->searchIds($languageCriteria, Context::createDefaultContext())->getIds();
+
+        if (\count($languageIds) !== 2) {
```

Comment:
```suggestion
        if (\count($languageIds) < 2) {
```

it should not skip the test, if there are more than two languages

Reply:
and maybe instead of skipping, let it fail? otherwise the test would not make sense

Reply:
> it should not skip the test, if there are more than two languages

It does not, the criteria has set a limit of 2. 

(From my point of view the crash appeared with two languages, so I don't think there is any win to test with more than two). 

> and maybe instead of skipping, let it fail? otherwise the test would not make sense

Amm I mean maybe, I just followed the pattern from old `generateSeoUrls`, which rn is renamed to `generateCategorySeoUrls` and the pre-existing `markTestSkipped` lines were just moved to `findRandomSalesChannel` since its both reused from `generateCategorySeoUrls` and `generateProductSeoUrls`.

(The other `markTestSkipped` would skip test if it did not find any sales channel in test db).

So I guess we should either change both or none. WIll wait for your comment.

Reply:
ah right the limit is set to two. I have overseen that :see_no_evil: 

if it was like this before, we can leave it like this